### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lbluque**
Fixed a counting error in normalize_nf and fixed unicode_literals error caused by python 3 support

### CODE REVIEW

**Comment by lbluque** [comment]
Yes, completely unnecessary. removed.

File: tomopy/misc/morph.py [hunk 1]
```diff
@@ -73,7 +73,7 @@
 LIB_TOMOPY = extern.c_shared_lib('libtomopy')
 
 
-def pad(arr, axis, npad=None, mode='constant', **kwargs):
+def pad(arr, axis, npad=None, mode=str('constant'), **kwargs):
```
Comment (tacaswell) [inline tomopy/misc/morph.py]: Why add the `str` here?  There should be no reason for that.

**Comment by tacaswell** [comment]
Ln 108 should probably use `six.text_type`

File: tomopy/misc/morph.py (hunk 1 above)
Comment (lbluque) [inline tomopy/misc/morph.py]: So the unicode_literals import from future causes makes the mode kwarg be of type unicode, and this breaks the numpy pad function when using python 2.7
But I agree that this really should not be there...

Comment (tacaswell) [inline tomopy/misc/morph.py]: It would read better to put the `str` in the call to pad rather than in the signature.

Comment (lbluque) [inline tomopy/misc/morph.py]: Right, reads way better!

**Pull request merged**